skimage/transform/radon_transform.py

```diff
@@ -191,7 +193,7 @@ def iradon(radon_image, theta=None, output_size=None,
         between 0 and 180 (if the shape of `radon_image` is (N, M)).
     output_size : int, optional
         Number of rows and columns in the reconstruction.
-    filter : str, optional
+    filter_name : str, optional
```

Comment:
I think we are trying to add notes, that something has been removed in a specific version. Versionchanged or versionadded